python-package/xgboost/dask.py

```diff
@@ -1719,7 +1719,7 @@ def _client_sync(self, func: Callable, **kwargs: Any) -> Any:
     """Implementation of the Scikit-Learn API for XGBoost.""", ["estimators", "model"]
 )
 class DaskXGBRegressor(DaskScikitLearnBase, XGBRegressorBase):
-    # pylint: disable=missing-class-docstring
+    """dummy doc string to workaround pylint, replaced by the decorator."""
```

Comment:
For some reason, the latest pylint ignores the comment. The workaround can be verified by visiting the latest doc: https://xgboost--7967.org.readthedocs.build/en/7967/python/python_api.html#xgboost.dask.DaskXGBRegressor .